modules/tinymce/src/plugins/visualchars/main/ts/api/Settings.ts

```diff
@@ -9,6 +9,9 @@ import Editor from 'tinymce/core/api/Editor';
 
 const isEnabledByDefault = (editor: Editor) => editor.getParam('visualchars_default_state', false);
 
+const hasForcedRootBlock = (editor: Editor): boolean => editor.getParam('forced_root_block', false) !== false;
```

Comment:
This default is incorrect here. It shouldn't have a default based on the previous code.

Reply:
That was me compensating for something I thought I saw but didn't. It's good you're here to catch these mistakes.